samples/sample_multi_transcode/src/sample_multi_transcode.cpp

```diff
@@ -387,7 +387,7 @@ void Launcher::DoTranscoding()
             if (!m_pThreadContextArray[i]->handle.valid())
                 continue;
 
-            auto waitSts = m_pThreadContextArray[i]->handle.wait_for(std::chrono::milliseconds(1));
+            auto waitSts = m_pThreadContextArray[i]->handle.wait_for(std::chrono::milliseconds(66));
```

Comment:
Add a comment above. Basically repeat my explanation about payslip interval to check the state of working threads.

Reply:
done